analyzers/tests/SonarAnalyzer.Test/TestCases/ConditionalStructureSameImplementation_If.cs

```diff
@@ -93,6 +93,35 @@ public void Test_Overloads()
                 foo = IntExtension.FooInt(foo);
             }
         }
+
+        // https://github.com/SonarSource/sonar-dotnet/issues/1255
+        public void ExceptionOfException(int a)
```

Comment:
Let's add also a test case exactly like this one but with an `else` clause at the end.